File: 3_conversao_de_temperatura.c
#include <stdio.h>
int main()
{
  float celsius, fahrenheit;

  printf("Digite a temperatura em graus Celsius:");
  scanf("%f", &celsius);

  fahrenheit = (celsius*1.8)+32;

  printf("Graus em Fanrenheit: %.2f", fahrenheit);

  return 0;
}

File: Java/3_conversao_de_temperatura.java
import java.util.Scanner;

public class ConversorDeTemperatura {
  public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);

    System.out.print("Digite a temperatura em graus Celsius: ");
    double celsius = scanner.nextDouble();

    double fahrenheit = (celsius * 1.8) + 32;

    System.out.println("Graus em Fahrenheit: " + fahrenheit);
  }
}